plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -469,55 +469,29 @@ class read_only {
    bool get_primary_key_internal(name code, name scope, name table, uint64_t primary_key, row_requirements require_table,
                                  row_requirements require_primary, Function&& f) const {
       const auto db_backing_store = get_backing_store();
```

Comment:
Go ahead and remove backing_store concept.

Reply:
done